play/process-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "0.5"
```

Comment:
Cool, didn't know about process-compose before.

Reply:
It's nice, although I had some issues with `depends_on` chains not working properly. But the checked in code here does not have this.